addons/sale_expense/tests/test_sale_expense.py

```diff
@@ -46,6 +46,7 @@ def test_sale_expense(self):
             'sheet_id': sheet.id,
             'sale_order_id': so.id,
         })
+        exp.write({'unit_amount': 621.54})
```

Comment:
The unit_amount shouldn't be erase on the expense creation IMO. 

I think that this modification is actually hiding a real issue.

Reply:
Indeed, there was an issue :sweat_smile:  I changed stuff so this is not needed anymore.